src/views/NewQuestion/Sections/SectionForm.tsx

```diff
@@ -138,11 +138,13 @@ const SectionForm = (props: RouteComponentProps) => {
         {isDragActive ? (
           <p>Drop the files here ...</p>
         ) : (
-          <span className="message-text">
+          <React.Fragment>
             <button className="upload">+</button>
```

Comment:
Mulig å bare legge til en `<div className="tooltip">` her, for GDPR-hensyn? Akkurat den samme som er brukt i ChatInput.tsx, med teksten "Hvis du sender et vedlegg, må du gjerne fjerne navnet ditt eller andre ting fra dokumentet som kan identifisere deg."

<img width="362" alt="Skjermbilde 2019-12-11 kl  12 40 09" src="https://user-images.githubusercontent.com/11062132/70618537-6bfe3680-1c13-11ea-847d-dd32092b039c.png">

Reply:
Siden det ikke er laget som en modul, ble det strevsomt å prøve å legge det inn i CSS-en. Oppretter en task på det 😄